doc/reference/trd-legal.md

```diff
@@ -0,0 +1,188 @@
+Licensing and Copyrights
+========================================
+
+**TRD:** <br/>
+**Working Group:** Core<br/>
+**Type:** Documentary<br/>
+**Status:** Draft <br/>
+**Author:** Pat Pannuto<br/>
+**Draft-Created:** 2022/11/05 <br/>
+**Draft-Modified:** 2022/11/05 <br/>
+**Draft-Version:** 1 <br/>
+**Draft-Discuss:** tock-dev@googlegroups.com<br/>
+
+Abstract
+--------
+
+This document describes Tock’s policy on licensing and copyright. It explains
+the rationale behind the license selection (dual-license, MIT or Apache2) and
+copyright policy (optional, and authors may retain copyright if desired). It
+further outlines how licensing and copyright shall be handled throughout
+projects under the Tock umbrella.
+
+Explicitly not discussed in this TRD are issues of trademark, the Tock brand,
+logos, or other non-code assets and artifacts.
+
+
+1. Introduction
+===============
+
+Tock’s goal is to provide a safe, secure, and efficient operating system for
+microcontroller-class devices. The Tock project further believes it is
+important to enable and support widespread adoption of safe, secure, and
+efficient software. Tock also seeks to be an open and inclusive project, and
+Tock welcomes contributions from any individuals or entities wishing to
+improve the safety, security, reliability, efficiency, or usability of Tock
+and the Tock ecosystem.
+
+The intent of these policies is to best satisfy the needs of all stakeholders
+in the Tock ecosystem.
+
+
+2. Licensing
+============
+
+Tock’s primary licensing goal is to ensure ease of use for Tock project elements.
+For this reason, Tock offers its resources under the minimally encumbered MIT
+license and under the Apache2 license for those entities concerned about possible
+litigation issues.
+
+All software artifacts under the umbrella of the Tock project are
+dual-licensed as Apache License, Version 2.0 (LICENSE-APACHE or
+http://www.apache.org/licenses/LICENSE-2.0) or MIT license (LICENSE-MIT or
+http://opensource.org/licenses/MIT).
+
+All contributions to Tock projects MUST be licensed under these terms.
+
+
+3. Copyright
+============
+
+Entities contributing resources to open-source projects often require attribution
+to recognize their efforts. Copyright is a common means to provide this. For
```

Comment:
Was this supposed to say "Copyright notices are a common means to provide this."? "Copyright" is a much broader term than "Copyright notices".